plugin/omero_iviewer/iviewer_settings.py

```diff
@@ -25,6 +25,13 @@
 # load settings
 IVIEWER_SETTINGS_MAPPING = {
 
+    "omero.web.iviewer.max_projection_bytes":
+        ["MAX_PROJECTION_BYTES",
+         1024 * 1024 * 256,
+         int,
+         ("Maximum bytes of raw pixel data allowed for Z-projection. "
+          "Above this threshold, Z-projection is disabled")],
```

Comment:
Maybe indicate that this value will be ignored if the limit is greater that the one set server-side.
Server side is work-in-progress but when it is in-place that will be the rule.

Reply:
Seems wrong for the documentation to describe *future* functionality. 
Since I'll need to open a PR to update the functionality, I can change the description at the time to match (depending on what we end up with).

Reply:
we can review the wording later on when we implement the size limit from server
My point is that if people start setting a high limit, this will be ignored later on and will default to the server one. This might come at a surprise for that.